generators/java/reactor-netty/expected-client/src/test/java/com/example/routes/SimpleClientTest.java

```diff
@@ -44,7 +46,7 @@ public void testPost() {
     @Test
     @Override
     public void testGet() {
-        Mono<Dto> response = simpleClient.simpleGet("idValue", "param1Value", Param2OfSimpleGet.VALUE1);
+        Mono<Dto> response = simpleClient.simpleGet(LocalDate.now(), "idValue", "param1Value", Param2OfSimpleGet.VALUE1);
```

Comment:
minor. It's better to constant values in tests (not random or current time-based)